float64 phi
float64 theta
float64 psi
float64 thrust

int8 controlflag
